commands/handbook.py

```diff
@@ -0,0 +1,71 @@
+from commands.base import Command
+from discord import Embed
+from helpers import CommandFailure
+
+import bs4, re, requests
+
+
+def subject_details(code):
+    code = code.upper()
+
+    page = requests.get('https://www.handbook.unsw.edu.au/undergraduate/courses/2019/' + code)
+
+    if page.status_code != 200:
```

Comment:
There's a helper function for this in helpers.py called `is_good_response()`, consider using that instead.